R/write.R

```diff
@@ -743,7 +743,6 @@ write_data_table <- function(
 
   ## Input validating
   assert_workbook(wb)
-  if (missing(x)) stop("`x` is missing")
```

Comment:
This changes `write_data()` which is deprecated, but might still be used? But it's fine for me, too.

Reply:
the evil me thinks this is fine, because if reported, we can point to the non-deprecated solution ;)!

Reply:
Hehe, that's one way to look at it